src/stores/logFileStore.ts

```
@@ -0,0 +1,166 @@
+import {SEARCH_PARAM_NAMES} from "../typings/url";
+import {
+    CursorType,
+    FileSrcType,
+} from "../typings/worker";
+import {getConfig} from "../utils/config";
+import useContextStore from "./contextStore";
+import useLogExportStore, {LOG_EXPORT_STORE_DEFAULT} from "./logExportStore";
+import useLogFileManagerProxyStore from "./logFileManagerProxyStore";
+import useQueryStore from "./queryStore";
+import useUiStore from "./uiStore";
+import useViewStore, {VIEW_STORE_DEFAULT} from "./viewStore";
```

ESLint

[error] 'VIEW_STORE_DEFAULT' is defined but never used.

(@typescript-eslint/no-unused-vars)